visualization/visualization_tools/src/marker_visualization.cpp

```diff
@@ -69,8 +72,9 @@ bool MarkerVisualization::createMarkers(rviz::DisplayContext* context, Ogre::Sce
 	Ogre::Vector3 pos;
 
 	try {
-		tf::TransformListener* tf = context->getFrameManager()->getTFClient();
 		std::string error_msg;
+#ifdef RVIZ_TF1
+		tf::TransformListener* tf = context->getFrameManager()->getTFClient();
 		if (!tf->canTransform(planning_frame_, fixed_frame, ros::Time(), &error_msg)) {
```

Comment:
Could you remove this `canTransform` check? Github forbids me to add a suggestion for it...
The block is not there for the new TF2 version, and it's not needed either because exceptions are caught below (`tf::*Exception` are plain typedefs for their `tf2::` counterparts)